packages/cli/test/integration/shared/testDb.ts

```diff
@@ -179,34 +188,40 @@ export async function saveCredential(
 }
 
 // ----------------------------------
-//          user creation
+//           user creation
 // ----------------------------------
 
-/**
- * Store a user in the DB, defaulting to a `member`.
- */
-export async function createUser(attributes: Partial<User> = {}): Promise<User> {
+export async function createFullUser(
+	attributes: Partial<User> & { globalRole: Role },
+): Promise<User> {
 	const { email, password, firstName, lastName, globalRole, ...rest } = attributes;
+
 	const user = {
 		email: email ?? randomEmail(),
-		password: hashSync(password ?? randomValidPassword(), genSaltSync(10)),
+		password: hashPassword(password ?? randomValidPassword()),
 		firstName: firstName ?? randomName(),
 		lastName: lastName ?? randomName(),
-		globalRole: globalRole ?? (await getGlobalMemberRole()),
+		globalRole,
 		...rest,
 	};
 
 	return Db.collections.User!.save(user);
 }
 
-export async function createOwnerShell() {
-	const globalRole = await getGlobalOwnerRole();
-	return Db.collections.User!.save({ globalRole });
+export async function createMemberShell(globalMemberRole: Role): Promise<User> {
+	if (globalMemberRole.scope !== 'global' || globalMemberRole.name !== 'member') {
+		throw new Error(`Invalid role received: ${JSON.stringify(globalMemberRole)}`);
+	}
+
+	return Db.collections.User!.save({ globalRole: globalMemberRole });
```

Comment:
I thought of this, but I decided to force the caller to pass in the role to prevent them from triggering a query that should always be unnecessary, since all the suites from now make the roles available at the top level. Let me know otherwise.